src/renderer/store/configure.ts

```diff
@@ -46,10 +46,8 @@ if (isLogConsoleEnabled || isLogFileEnabled) {
   middlewares.push(createReduxLogger(loggerConfig));
 }
 
-const createStoreWithMiddleware = applyMiddleware(...middlewares)(createStore);
-
 export default function configureStore(initialState?) {
-  const store = createStoreWithMiddleware(rootReducer, initialState);
+  const store = createStore(rootReducer, initialState, applyMiddleware(...middlewares));
```

Comment:
No need to do what we were doing before, `createStore` accepts a third argument for middlewares.